docs/src/TerminalVelocityComparisons.jl

```diff
@@ -1,37 +1,39 @@
 import Plots as PL
 
-FT = Float64
-
 import CloudMicrophysics as CM
 import CLIMAParameters as CP
 
+FT = Float64
+
 const CMT = CM.CommonTypes
 const CM1 = CM.Microphysics1M
 const CM2 = CM.Microphysics2M
 const CMP = CM.Parameters
 
-include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
-toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
-const param_set = cloud_microphysics_parameters(toml_dict)
-
 const rain = CMT.RainType()
+const liquid = CMT.LiquidType()
+const ice = CMT.IceType()
+const snow = CMT.SnowType()
 const SB2006 = CMT.SB2006Type()
 const Chen2022 = CMT.Chen2022Type()
+const APS = CMP.AbstractCloudMicrophysicsParameters
+
+include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const param_set = cloud_microphysics_parameters(toml_dict)
 
 """
-    rain_terminal_velocity_individual_C(ρ, scheme, D_r)
+    rain_terminal_velocity_individual_Chen(param_set, ρ, D_r)
 
- - `ρ`` - air density
- - `scheme` - type for parameterization
+ - `param_set` - set with free parameters
+ - `ρ` - air density
  - `D_r` - diameter of the raindrops
 
- Returns the fall velocity of a raindrop using multiple gamma-function terms
- to increase accuracy for `scheme == Chen2022Type`
+Returns the fall velocity of a raindrop from Chen et al 2022
 """
 function rain_terminal_velocity_individual_Chen(
```

Comment:
It would be nice to have the function throw a warning or error if D_r >= 0.1mm. If you do end up doing that, make sure to add a gpu/unit test for it